src/hooks/useAppUpdate.ts

```diff
@@ -0,0 +1,97 @@
+import { logger } from "config/logger";
+import { useDebugStore } from "ducks/debug";
+import { useRemoteConfigStore } from "ducks/remoteConfig";
+import { isIOS } from "helpers/device";
+import { isDev } from "helpers/isEnv";
+import {
+  isVersionBelowLatest,
+  isVersionBelowRequired,
+} from "helpers/versionComparison";
+import useAppTranslation from "hooks/useAppTranslation";
+import { useToast } from "providers/ToastProvider";
+import { useCallback } from "react";
+import { Linking } from "react-native";
+import { getBundleId, getVersion } from "react-native-device-info";
+
+const IOS_BUNDLE_ID = "id6743947720";
```

Comment:
```suggestion
const IOS_APPLE_ID = "id6743947720";
```

just to reduce confusion as Bundle Id actually is `org.stellar.freighterwallet` but the Apple ID is `6743947720`